internal/operands/common-templates/reconcile.go

```diff
@@ -87,6 +95,32 @@ func (c *commonTemplates) Cleanup(request *common.Request) error {
 	return nil
 }
 
+func pauseCRs(request *common.Request) {
+	patch := []byte(`{"metadata":{"annotations":{"kubevirt.io/operator.paused": "true"}}}`)
+	var kubevirtCommonTemplatesBundles kvsspv1.KubevirtCommonTemplatesBundleList
+	err := request.Client.List(context.TODO(), &kubevirtCommonTemplatesBundles, &client.ListOptions{})
+	if err != nil {
+		request.Logger.Error(err, fmt.Sprintf("Error listing common template bundles: %s", err))
+		return
+	}
+	if err == nil && len(kubevirtCommonTemplatesBundles.Items) > 0 {
```

Comment:
This check can be removed. At this point `err` is always `nil`, and it is not needed to check a slice's size before looping over it.